src/arrayMethodJoin.js

```diff
@@ -4,8 +4,26 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let likeJoin = '';
+    const sep = separator;
+
+    if (this.length === 0) {
+      return '';
+    }
```

Comment:
you will anyway return '' if we have falsy value like 0. Because you have return statement return likeJoin; at 26th row
```suggestion

```